.gitignore

```diff
@@ -156,3 +156,6 @@ venv
 # Vim
 .*.swp
 *.swp
+
+# tools
+tools/prometheus*
```

Comment:
Is this to ignore the downloaded prometheus server? If so, we can drop this, and download it to a temp dir.

Reply:
If we download to temp dir, there is no good way for users to start the server.

`./prometheus --config.file=prometheus.yml   # start your Prometheus server.`